Client/Frontend/Browser/LoginsHelper.swift

```diff
@@ -228,7 +228,7 @@ class LoginsHelper: TabHelper {
             }
 
             let json = JSON(jsonObj)
-            let src = "window.__firefox__.logins.inject(\(json.rawString()))"
+            let src = "window.__firefox__.logins.inject(\(json.rawString()!))"
```

Comment:
@justindarc Is this the right thing to do? If `json.rawString()` returns `nil`, this will crash. Why does a force unwrap fix this regression? Should we not wrap this in an `if let` if `rawString()` returns an optional?